.yamato/build_and_test_linux_x64.yml

```diff
@@ -23,4 +24,5 @@ artifacts:
   linux64:
     paths:
       - artifacts/bin/coreclr/**
```

Comment:
do we still need this? I added this blindly at first.

Reply:
similar with the `artifacts/repro/**` line I don't recall adding that one though.

Reply:
Probably not. I left as it was there originally. I can remove.